app/controllers/posts_controller.rb

```diff
@@ -4,7 +4,27 @@ def index
     @posts = Post.where(author_id: params[:user_id])
   end
 
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(posts_params)
+    @post.author = current_user
```

Comment:
- [ ] this piece of code should be written like this:
```
@post = @current_user.posts.new(post_params)
```

Reply:
Thanks for you review. It is very appreciated. I think this is more like an optional suggestion being that the lines of code is more of a suggestion on how to write a one-liner of the same thing i wrote. However, I really appreciate you review and will do as requested. 

Cheers 😄